cpuid.go

```diff
@@ -231,27 +241,14 @@ var flagNamesArm = map[ArmFlags]string{
 	GPA:      "GPA",      // Generic Pointer Authentication
 }
 
-// x86 Advanced Matrix Extensions features, in CPUInfo.AmxFeatures
-const (
-	AMXBF16 AmxFlags = 1 << iota // Tile computational operations on BFLOAT16 numbers
-	AMXTILE                      // Tile architecture
-	AMXINT8                      // Tile computational operations on 8-bit integers
-)
-
-var flagNamesAmx = map[AmxFlags]string{
-	AMXBF16: "AMXBF16", // Tile computational operations on BFLOAT16 numbers
-	AMXTILE: "AMXTILE", // Tile architecture
-	AMXINT8: "AMXINT8", // Tile computational operations on 8-bit integers
-}
-
 // CPUInfo contains information about the detected system CPU.
 type CPUInfo struct {
 	BrandName      string   // Brand name reported by the CPU
 	VendorID       Vendor   // Comparable CPU vendor ID
 	VendorString   string   // Raw vendor string.
-	Features       Flags    // Features of the CPU (x64)
+	Features       Flags    // Features of the CPU (x64) (deprecated)
+	flagSet        FlagSet  // Features of the CPU
```

Comment:
I was a bit distracted by the name `flagSet` here as it's overloaded with the `flags` package. What do you think about renaming it? `features2` or `featureSet`, for example((?)

Reply:
This is in PR #59. Fixed there in the latest update. (This was well spotted, naming looks better now!)